rtgui/colorappearance.h

```diff
@@ -29,12 +29,12 @@
 #include "colorprovider.h"
 
 class ColorAppearance final :
-    public ToolParamBlock,
-    public AdjusterListener,
-    public FoldableToolPanel,
-    public rtengine::AutoCamListener,
-    public CurveListener,
-    public ColorProvider
+        public ToolParamBlock,
```

Comment:
@Beep6581 This is wrong. Maybe `astyle` isn't aware of the C++11 keyword `final`.